app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
```

Comment:
Hmm, right definitely should think about that. The way I had originally thought it through is that there would be two types of events...

public, which everyone receives (we could just fan those out to all sockets)
private, for which the intended recipients should be in the payload (and, depending on the state of the game, which is in memory, more may be added to that list, e.g. wire taps)

but I think you're right, it'd be a whole lot easier, especially in the case of a disconnect, if we just stored the recipients of each event.